app/lobby/lobby.tsx

```
@@ -0,0 +1,211 @@
+    // navigation logic here
+  };
+
+  const chessSymbols = ['♔', '♕', '♖', '♗', '♘', '♙'];
+
+  return (
+    <View style={styles.container}>
+      {/* Floating background pieces */}
+      <View style={StyleSheet.absoluteFill}>
+        {chessSymbols.map((piece, index) => {
+          const style = useAnimatedStyle(() => ({
+            transform: [{ translateY: floatingPieces[index].translateY.value }],
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)